devito/ops/transformer.py

```diff
@@ -37,7 +42,44 @@ def opsit(trees, count):
         parameters
     )
 
-    return ops_kernel
+    stencil_arrays_initializations = itertools.chain(*[
+        to_ops_stencil(p, node_factory.ops_args_accesses[p])
+        for p in parameters if isinstance(p, OpsAccessible)
+    ])
+
+    pre_time_loop = stencil_arrays_initializations
+
+    return pre_time_loop, ops_kernel
+
+
+def to_ops_stencil(param, accesses):
+    dims = len(accesses[0])
+    pts = len(accesses)
+    stencil_name = "s%sd_%s_%spt" % (dims, param.name, pts)
```

Comment:
put in ops/utils.py:
namespace['stencil_name'] = lambda dims, name, pts: 's%sd_%s_%spt' % (dims, name, pts)
and replace 
stencil_name = "s%sd_%s_%spt" % (dims, param.name, pts)
to
stencil_name = namespace['stencil_name'](dims, param.name, pts)